### PR TITLE
Add new clicklabel class to qt makefile, remove old .png's

### PR DESCRIPTION
**Pull request by sedited**
Adds the ClickLabel class to the qt makefile recipe, removed now unused
.png icon targets.

### CODE REVIEW

File: src/Makefile.qt.include [hunk 1]
```diff
@@ -236,27 +236,19 @@ RES_ICONS = \
...
   qt/res/icons/editcopy.png \
   qt/res/icons/editpaste.png \
   qt/res/icons/ex.png \
   qt/res/icons/export.png \
   qt/res/icons/filesave.png \
-  qt/res/icons/history.png \
   qt/res/icons/gold_cpumined.gif \
-  qt/res/icons/gold_cpumined.png \
   qt/res/icons/key.png \
-  qt/res/icons/lock_closed.png \
```
Comment (denravonska) [inline src/Makefile.qt.include:251]: Aren't these still used?

> qt/coincontroldialog.cpp:    contextMenuItem->setIcon(COLUMN_CHECKBOX, QIcon(":/icons/lock_closed"));
> qt/coincontroldialog.cpp:                itemOutput->setIcon(COLUMN_CHECKBOX, QIcon(":/icons/lock_closed"));

Comment (sedited) [reply src/Makefile.qt.include:251]: That portion of code is commented out in coincontroldialog.cpp . Why is it commented out @skcin  ?
Edit: In other files the lock_closed is changed correctly to .svg + whatever theme is used.

Comment (sedited) [reply src/Makefile.qt.include:251]: Actually, checking the blame it looks like it was never used in the first place.

Comment (denravonska) [reply src/Makefile.qt.include:251]: Aye, it's used in more places like `bitcoingui.cpp` though.

Comment (sedited) [reply src/Makefile.qt.include:251]: I can only see the .svg files in the bitcoingui.cpp , where /icons/lock_closed_"+sSheet imports the correctly styled .svg . Do you have a line number?

**Pull request merged**